pkg/config/config.go

```diff
@@ -44,6 +44,7 @@ func (c Config) GetProvider(name string) (*Provider, error) {
 type CloudQuery struct {
 	PluginDirectory string            `hcl:"plugin_directory,optional"`
 	PolicyDirectory string            `hcl:"policy_directory,optional"`
+	DBPersistence   bool              `hcl:"db_persistence,optional"`
```

Comment:
So in the RFC we had a new `policy` which enabled us to name the attribute `db_persistence` because the context was only about policies. Is there a reason why we can't do that here?